### PR TITLE
Move architecture overview image to concept explanation page

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

In the current concept explanation page, we have definitions of PipeCD's components such as: piped, control-plane, etc. but do not have an image to explain what it is in a big picture so it's hard to follow, the image which shows PipeCD architecture overview is on the overview page, which contains stub not really related to the thing shown in the image. So my idea is to move the architecture overview image to the concepts explanation page and use a different image to show what PipeCD is on the overview page as below:

Overview:
Screen Shot 2022-02-21 at 12 24 51

Concepts:
Screen Shot 2021-12-29 at 22 29 15

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nghialv** [comment]
Nice idea!
I guess we could have a better image so please give me time to think about this.

**Comment by khanhtc1202** [comment]
Sure, thank you 👍 
/hold

**Comment by khanhtc1202** [comment]
Updated the explanation image, PTAL, when you guys have time 😉 
/hold cancel

**Comment by khanhtc1202** [comment]
Thanks @nghialv @knanao, I updated the explanation image, PTAL 🙏

**Comment by nghialv** [comment]
Thank you.
/lgtm

**Comment by knanao** [comment]
TYVM :D
/approve
